### ISSUE TITLE
Power argument in geometry.pointcloud.PointCloud appears to be broken

### ISSUE THREAD

**Issue by JTT94** [ott-jax/ott#152]
When setting power =/= 2, Sinkhorn appears to stop after 0 iterations but has not converged

https://colab.research.google.com/drive/1oXRZsGqVoz7aTQXraQqQ0Xdvt_dOV2qh?usp=sharing

<img width="710" alt="image" src="https://user-images.githubusercontent.com/28487211/195437427-0d4bf69b-ae8e-4ff9-a46a-aa6c04b61856.png">

**Comment by marcocuturi** [comment]
this seems like a 0^1 type of problem
here is what i get for `power=1.0` in your exemple (weirdly enough `power=3` has nans in exactly the same places...)
<img width="875" alt="image" src="https://user-images.githubusercontent.com/22635213/195440514-036065ca-19d5-4308-9825-5cabdffd8cc0.png">

**Comment by JTT94** [comment]
I think you're right, this hack seems to work
```
geom_full = pointcloud.PointCloud(X, X+0.0001, epsilon=epsilon, power=1.0)
out = run_sinkhorn(geom_full, a, b, f_init=None)
jnp.sum(out.errors>-1)
```
DeviceArray(1141, dtype=int32)

**Comment by marcocuturi** [comment]
fixed with #153
